stream-chat-android-offline/detekt-baseline.xml

```diff
@@ -132,6 +132,8 @@
     <ID>NestedBlockDepth:FilterObjectConverter.kt$private fun Map.Entry&lt;String, Any>.toFilterObject(): FilterObject</ID>
     <ID>NestedBlockDepth:UploadAttachmentsWorker.kt$UploadAttachmentsWorker$suspend fun uploadAttachmentsForMessage( channelType: String, channelId: String, messageId: String, ): Result&lt;Unit></ID>
     <ID>PrintStackTrace:UploadAttachmentsWorker.kt$UploadAttachmentsWorker$e</ID>
+    <ID>ReturnCount:Channel.kt$ internal fun Channel.addMember(member: Member?): Channel</ID>
```

Comment:
Why do we need to add those two methods here/

Reply:
Because of having several `return` inside the method.
Dunno why it's restricted.

Reply:
@JcMinarro what do you think?